pkg/services/publicdashboards/service/service_test.go

```diff
@@ -1335,10 +1357,9 @@ func TestDeletePublicDashboard(t *testing.T) {
 				serviceWrapper: serviceWrapper,
 			}
 
-			err := service.Delete(context.Background(), "uid")
+			err := service.Delete(context.Background(), "pubdashUID", "uid")
 			if tt.ExpectedErrResp != nil {
-				assert.Equal(t, tt.ExpectedErrResp.Error(), err.Error())
-				assert.Equal(t, tt.ExpectedErrResp.Error(), err.Error())
+				assert.Error(t, tt.ExpectedErrResp, err)
```

Comment:
This assert is not evaluating equality, it's only confirming that the `tt.ExpectedErrResp` is an error (which always be because above we are asking if it is not nil). The `err` parameter here is used for messages. Maybe you should use `assert.Equal` to confirm you are receiving the error you expect.

Reply:
done, changed